### DISCUSSION TITLE
Is there a way to disable `width=variable` only for git blame ?

### DISCUSSION THREAD

**Discussion by bew** [dandavison/delta#1094] [Q&A] [answered]
I'm trying out the option `width = variable`, I think it's interesting for diffs, but for `git blame some/file` it's not great, as the colored lines for commits ages are better when full-width.

Is there any way to disable the option only for `git blame` processing ?

**Comment by Kr1ss-XD** [comment, marked as answer]
Hi @bew,

you could make use of the `pager.blame` option in the `gitconfig`, and set that so that `delta` would be called with `--width=0` when running `git blame`, and have a default of `--width=variable` for any other git subcommand : 

```gitconfig
[pager]
	blame	=	delta --width=0
	diff	= 	delta
	log		=	delta
	; ...
[delta]
	width	=	variable
	; ...
```

**Reply by bew** [reply]
Thanks :)
